starter-code/index.html

```diff
@@ -1,31 +1,60 @@
-<!DOCTYPE html>
+  <!DOCTYPE html>
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" type="text/css" href="css/main.css">
 </head>
 <body>
-  Instagram
-
-  Capture and Share the World's Moments
-
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
-
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
-
-  Oh yeah, did we mention it's free?
-
-  Download on the App Store
-  Get it on Google Play
-
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
-
-  &copy; 2014 Instagram
+  <div class="main-container">
+    <div class="top">
+      <div class="phone-picture">
+        <img src="images/phones.png" alt="Phones">
+      </div>
+      <div class="logo-login">
+        <div id="logo">
+          <img src="images/brand.png">
+        </div>
+        <div id="login">
+          <button type="button">Log in</button>
+        </div>
+      </div>
+    </div>
+    <div class="bottom">
+      <div class="frame">
+        <div id="header">
+          <h2>Capture and Share the World's Moments</h2>
+        </div>
+        <div id="text">
+          <p>Instagram is a <strong>fast, beautiful</strong> and <strong>fun</strong> way to share your life with friends and family.</p>
+          <p>Take a picture or video, choose a filter to transform its look and feel,
+          then post to Instagram &mdash; it's that easy. You can even share to Facebook,
+          Twitter, Tumblr and more. It's a new way to see the world.</p>
+          <p>Oh yeah, did we mention it's free?</p>
+        </div>
+        <div id="badge">
+          <ul>
+  				<li><a href="#"> <img src="images/badge-iphone.png" alt="App Store"/></a></li>
+          <li><a href="#"> <img src="images/badge-android.png" alt="Google Play"/></a></li>
+  			</ul>
+        </div>
+      </div>
+      <div class="table">
+        <table>
```

Comment:
For the most part, you want to avoid using tables for navigation. This would instead look better in an unordered list.